client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
+/**
+ * Adds the ability to trigger a button click event using a keyboard shortcut
+ * declared on the controlled element.
+ *
+ * @example
+ * ```html
+ * <button type="button" data-controller="w-kbd" data-w-kbd-key="[">Trigger me with the <kbd>[</kbd> key.</button>
+ * ```
+ */
+export class KeyboardController extends Controller<HTMLButtonElement> {
```

Comment:
```suggestion
export class KeyboardController extends Controller<HTMLButtonElement HTMLAnchorElement> {
```

While we're at it we probably should explicitly support links and buttons. 

This will ensure that `this.element` is typed as a link or a button.